<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>DialogHWBreakpoints</class>
 <widget class="QDialog" name="DialogHWBreakpoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hardware Breakpoints</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>BP 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtBP1">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cmbType1">
     <item>
      <property name="text">
       <string>Execute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Write</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Read/Write</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cmbSize1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1 Byte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Bytes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="chkBP1">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>BP 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtBP2">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cmbType2">
     <item>
      <property name="text">
       <string>Execute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Write</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Read/Write</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="cmbSize2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1 Byte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Bytes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="chkBP2">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>BP 3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtBP3">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cmbType3">
     <item>
      <property name="text">
       <string>Execute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Write</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Read/Write</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="cmbSize3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1 Byte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Bytes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCheckBox" name="chkBP3">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>BP 4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="txtBP4">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cmbType4">
     <item>
      <property name="text">
       <string>Execute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Write</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Read/Write</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="cmbSize4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1 Byte</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Bytes</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="chkBP4">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogHWBreakpoints</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogHWBreakpoints</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
